flamingo/scripts/dummy.py

```diff
@@ -0,0 +1,3 @@
+"""
+Dummy
+"""
```

Comment:
Is there a particular reason that this exists?

Reply:
This dummy was aded as a hacky work around to produce several plots with the same script. That is, we read the particle data only once, make all the calculations (x-ray, entropy, etc) and produce all the plots. Then call the  dummy script in the pipeline config.yml to properly link the plots and names in the generated webpage.

Reply:
The dummy script was removed in https://github.com/jemme07/pipeline-configs/pull/1, since @EvgeniiChaikin implemented a feature that allows creating multiple figures with a single script (https://github.com/SWIFTSIM/pipeline/pull/26).